packages/ui/src/theme/tokens/components/index.ts

```diff
@@ -51,6 +52,7 @@ import {
 export interface ComponentTokens {
   alert: AlertTokens;
   authenticator: AuthenticatorTokens;
+  backdrop: BackdropTokens;
```

Comment:
I think so too, but for reasons I mentioned above, I felt doing this would present the least amount of friction if Backdrop were to become a primitive. But I'm open to refactoring this and scoping it solely to InApp if that is what you guys think is best.